src/LinuxTracing/LinuxTracingIntegrationTest.cpp

```diff
@@ -155,7 +156,19 @@ class BufferTracerListener : public TracerListener {
   }
 
   void OnCallstackSample(orbit_grpc_protos::CallstackSample callstack_sample) override {
+    static uint64_t next_callstack_id = 1;
+
+    // TODO: Maybe test it on the sender level?
+    orbit_grpc_protos::CaptureEvent event_interned_callstack;
+    orbit_grpc_protos::InternedCallstack* interned_callstack =
+        event_interned_callstack.mutable_interned_callstack();
+    uint64_t callstack_key = next_callstack_id++;
+    interned_callstack->set_key(callstack_key);
+    *interned_callstack->mutable_intern() = callstack_sample.callstack();
+    events_.emplace_back(std::move(event_interned_callstack));
```

Comment:
I'm confused about why callstacks are processed and interned here. This is supposed to test the behavior of `LinuxTracing` without introducing new events.